spec/formatting.md

```diff
@@ -151,8 +150,8 @@ when resolving the value of the _expression_.
 > the implementation could interpret the _placeholder_ `{$date}` as if
 > the pattern included the function `:datetime` with some set of default options.
 
-Else, if the _expression_ consists of a _literal_,
-its resolved value is defined by _literal resolution_.
+Else, the _expression_ must consist of a _literal_.
```

Comment:
"must" is a 2119 keyword. It's not used normatively here (and thus you have it as lower case), but it would be easy to avoid using it at all.

```suggestion
Else, the _expression_ consists of a _literal_.
```

The lack of "if" communicates that this is a terminal state.